tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -377,6 +385,24 @@ def setup_airgap_rancher(bastion_node, number_of_nodes=NUMBER_OF_INSTANCES):
     bastion_node.execute_command(
         kube_config+"/snap/bin/kubectl create namespace cattle-system && 10")
     create_tls_secrets(bastion_node)
+    bootstrap_password=""
+    if RANCHER_HELM_EXTRA_SETTINGS:
+        for setting in RANCHER_HELM_EXTRA_SETTINGS.split("--set "):
+            settings_keypair = RANCHER_HELM_EXTRA_SETTINGS.split("=")
+            if "bootstrapPassword" in settings_keypair[0]:
+                bootstrap_password = settings_keypair[1]
+                print ("bootstrap password: ", bootstrap_password)
+                execution = bastion_node.execute_command(
+                    kube_config+
+                    "/snap/bin/kubectl create secret" +
+                    " generic -n cattle-system " +
+                    "bootstrap-secret --from-literal=bootstrapPassword=" +
+                    bootstrap_password + " && 10")
+                print(execution[0], execution[1])
+                assert execution[0].find("create") > -1, \
```

Comment:
`find` returns -1 if it could not find the substring. Otherwise, it returns the index of the start of the substring. So in this case, > -1 is usually best since I'm not sure of the exact position it will be in (I think `create` is one of the last words in the returned string).

Reply:
Oh you're right I forgot this is the find behavior. Thanks for clarifying it makes sense.